nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,269 @@
+|[Config Contexts](../models/extras/configcontext.md)|Config contexts can be used to provide additional data that you can't natively store in Nautobot.|
+
+### Examples of Plugins Defining Additional Providers
+|Name|Summary|Related Plugin|
+|:--|:--|:--|
+|Backup Configs|Backup configuration data.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+|Intended Configs|Stores the intended configurations, this grabs Nautobot data and runs through Jinja Templates.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+|Jinja Templates|Repository that holds Jinja templates to be used to generate intended configs.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+
+## Repository Details
+
+|Field|Explanation|
+|:--|:--|
+|Name|User friendly name for the repo.|
+|Slug|Computer-friendly name for the repo. Auto-generated based on the `name` provided, but you can change it if you wish.|
+|Remote URL|The URL pointing to the Git repo. Current git url usage is limited to `http` or `https`.|
+|Branch|The branch in the Git repo to use. Defaults to `main`.|
+|Token|The token is a personal access token for the `username` provided.  For more information on generating a personal access token see the corresponding links below. **Token is the only option supported currently.**|
+|Username|The Git username that corresponds with the personal access token above. Note not required for GitHub, but is for GitLab.|
+|Provides|Resource type(s) provided by this Git repo.|
+
+- [GitHub Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
+- [GitLab Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
+- [Bitbucket Personal Access Token](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html)
+
+
+## Using Git Data Sources
+This section will focus on examples and use the `user-guide` branch on the demo-git-datasources repo: `https://github.com/nautobot/demo-git-datasource`.
+
+### Export Templates
+
+[Export Templates](../models/extras/exporttemplate.md) allow a user to export Nautobot objects based on a custom template.  Export templates can change over time depending on the needs of a user.  Allowing export templates to reference a Git repo makes managing templates easier.
+
+A template can be used to put objects into a specific format for ingestion into another system, tool, or report.  It is possible that different templates are needed depending on specific users or teams.  This can lead to sprawl of export templates.  To keep accurate templates synced with Nautobot the Git Data Sources extensibility feature can be used.
+
+#### Step 1
+Navigate to the Data Sources Git integration. **Extensibility -> Git Repositories**.
+
+![](./images/git-data-source_1.png)
```

Comment:
Best practice for accessibility would be to include appropriate alt text for each of these images, e.g.:

```suggestion
![Menu showing "Git Repositories" item](./images/git-data-source_1.png)
```